code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -19,6 +19,10 @@
 	prime()
 	..()
 
+/obj/item/weapon/grenade/acid_melt()
+	prime()

```

Comment:
same thing happens when grenade is burned, I thought I'd copy that behavior for acid.